picard/ui/scripteditor.py

```diff
@@ -517,6 +738,23 @@ def check_format(self):
             if not self.get_script():
                 raise ScriptCheckError("", _("The file naming format must not be empty."))
 
+    def confirmation_dialog(self, message_text):
+        """Display a confirmation dialog with Ok and Cancel buttons.
+
+        Args:
+            message_text (str): Message to display
+
+        Returns:
+            bool: True if Ok, otherwise False
+        """
+        dialog = QtWidgets.QMessageBox(QtWidgets.QMessageBox.Warning,
+                                       _('Confirm'),
```

Comment:
We should move first element to next line (with proper indentation) (as done elsewhere):

```python
dialog = QtWidgets.QMessageBox(
    QtWidgets.QMessageBox.Warning,
```